### DISCUSSION TITLE
Missing players in weekly look up

### DISCUSSION THREAD

**Discussion by nickmiltgen** [cwendt94/espn-api#366] [ESPN API]
For a given week, I'm pulling all players in the (football) league with the following and compiling in separate pandas dfs:

```
query = League(league_id=####, year=2022,swid='{####}',espn_s2='####')
home = pd.DataFrame(columns = ['PLAYER_KEY','PLAYER_NAME','2022_'+str(1)])
away = pd.DataFrame(columns = ['PLAYER_KEY','PLAYER_NAME','2022_'+str(1)])
fa = pd.DataFrame(columns = ['PLAYER_KEY','PLAYER_NAME','2022_'+str(1)])
box_scores = query.box_scores(week=1)
for matchup in box_scores:
    for p in matchup.home_lineup:
        player_row = [p.playerId,p.name,p.points] 
        home.loc[len(home)] = player_row
    for p in matchup.away_lineup:
        player_row = [p.playerId,p.name,p.points] 
        away.loc[len(away)] = player_row
free_agents = query.free_agents(size=1000,week=1)
for p in free_agents:
    player_row = [p.playerId,p.name,p.points]
    fa.loc[len(fa)] = player_row
```

It appears that players that underwent waiver activity (adds/drops) earlier this morning are not showing up in the dfs despite specifying week 1 in my query. Is this expected?

Thanks in advance your time! Love the package!

**Comment by cwendt94** [comment]
Thats a great question. I just checked to see how espn website is doing it and it seems like it will always get the latest free agents. The specified week is for the player stats, so projected points, number of points, matchup, etc. So seems like this is expected.
